### PR TITLE
controller: Log when we generate a MC

### PR DESCRIPTION
**Pull request by cgwalters**
I'd like a bit more visibility into what the controller is doing.
While I could `oc watch machineconfigs` too, I think this one is
worth a log message.

### CODE REVIEW

File: pkg/apis/machineconfiguration.openshift.io/v1/helpers.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package v1
 import (
 	"sort"
 
+	"github.com/golang/glog"
```
Comment (kikisdeliveryservice) [inline pkg/apis/machineconfiguration.openshift.io/v1/helpers.go]: accidental ide auto-add?

File: pkg/controller/render/render_controller.go [hunk 2]
```diff
@@ -427,6 +427,7 @@ func (ctrl *Controller) syncGeneratedMachineConfig(pool *mcfgv1.MachineConfigPoo
 	_, err = ctrl.mcLister.Get(generated.Name)
 	if apierrors.IsNotFound(err) {
 		_, err = ctrl.client.MachineconfigurationV1().MachineConfigs().Create(generated)
+		glog.Infof("Generated machineconfig %s from %d sources", generated.Name, len(configs))
```
Comment (jlebon) [inline pkg/controller/render/render_controller.go]: I like this. How about we also print the names of all the source configs?

Comment (abhinavdahiya) [reply pkg/controller/render/render_controller.go]: already logs here
https://github.com/openshift/machine-config-operator/pull/253/files#diff-9d1c255b5e10f6159f19c5fafa976458R413

**Comment by abhinavdahiya** [comment]
> I'd like a bit more visibility into what the controller is doing.
While I could oc watch machineconfigs too, I think this one is
worth a log message.

- Use verbosity levels.
- Use events for major actions

**Comment by cgwalters** [comment]
> Use events for major actions

Can we do that as a separate PR after https://github.com/openshift/machine-config-operator/pull/256 ?

File: pkg/controller/render/render_controller.go (hunk 2 above)
Comment (cgwalters) [reply pkg/controller/render/render_controller.go]: That one seems to happen though even if nothing changed; gets verbose.  I deleted it.

**Pull request merged**